### ISSUE TITLE
Update DNSSEC14

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster#1044]
The next time we make an update to DNSSEC14 we should also fix this:
* The tag NO_RESPONSE_DNSKEY should be renamed to NO_VALID_DNSKEY. This is because the current tag name indicates that no response was received but its semantics also include situations where response with a badly broken DNSKEY is received.

**Comment by matsduf** [comment]
We could find another name, but "NO_VALID_DNSKEY" gives the impression that there are DNSKEY, but no valid ones.

**Comment by mattias-p** [comment]
> "NO_VALID_DNSKEY" gives the impression that there are DNSKEY, but no valid ones.

Not really. INVALID_DNSKEY means that there is a DNSKEY that is invalid. NO_VALID_DNSKEY means that there is no valid DNSKEY. That's pretty straight forward.
